scripts/esm-support/loader.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from 'node:fs/promises';
+import { fileURLToPath } from 'node:url';
+import { parseArgs } from 'node:util';
+
+import parseCSS from 'css-parse';
+
+const { values: { loadFullStyles } } = parseArgs({
+  options: {
+    loadFullStyles: { type: 'boolean' },
+  },
+});
+
+export async function resolve(specifier, context, next) {
```

Comment:
this is a fake css loader, it's only needed for our esm test build, it does not get used anywhere else